convert2rhel/cert.py

```diff
@@ -52,9 +60,19 @@ def install(self):
         loggerinst.info("Installing RHEL certificate to the system.")
 
         try:
-            utils.mkdir_p(self._system_cert_dir)
-            shutil.copy(self._cert_path, self._system_cert_dir)
+            utils.mkdir_p(self._target_cert_dir)
+            shutil.copy(self._source_cert_path, self._target_cert_dir)
         except OSError as err:
             loggerinst.critical("OSError({0}): {1}".format(err.errno, err.strerror))
 
-        loggerinst.debug("Certificate copied to %s." % self._system_cert_dir)
+        loggerinst.debug("Certificate copied to %s." % self._target_cert_dir)
+
+    def remove(self):
+        """Remove certificate (.pem), which was copied to system's cert dir."""
+        loggerinst.info("Removing the certificate %s." % self._target_cert_path)
```

Comment:
```suggestion
        loggerinst.task("Rollback: Removing installed RHSM certificate")
```

Without this, the output is mixed up with the previous task:
```
[07/13/2021 10:37:09] TASK - [Rollback: Restoring /etc/yum/pluginconf.d/versionlock.list from backup] 
WARNING - /etc/yum/pluginconf.d/versionlock.list hasn't been backed up
Removing the certificate /etc/pki/product-default/69.pem.
[07/13/2021 10:37:09] DEBUG - Certificate removed
```